### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled and tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added regression tests
- [ ] added unit tests
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: docs/markdown/security/powerdns-advisory-2015-02.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 * CVE: CVE-2015-5230
 * Date: 2nd of September 2015
-* Credit: Pyry Hakulinen and Ashish Shukla at Automattic
+* Credit: Pyry Hakulinen and Ashish Shukla at Automatic
```
Comment (pieterlexis) [inline docs/markdown/security/powerdns-advisory-2015-02.md]: The company _is_ called Automattic 😄, so please remove this change.

File: docs/markdown/security/powerdns-advisory-2015-02.md [hunk 2]
```diff
@@ -26,5 +26,5 @@ contains a fix to this issue. A minimal patch is [available here](https://downlo
 
 This issue is entirely unrelated to [Security Advisory 2015-01](powerdns-advisory-2015-01.md)/CVE-2015-1868.
 
-We'd like to thank Pyry Hakulinen and Ashish Shukla at Automattic for finding and
+We'd like to thank Pyry Hakulinen and Ashish Shukla at Automatic for finding and
```
Comment (pieterlexis) [inline docs/markdown/security/powerdns-advisory-2015-02.md]: The company is called Automattic 😄, so please remove this change.

File: m4/boost.m4 [hunk 3]
```diff
@@ -408,7 +408,7 @@ dnl until we succeed to link conftest.o in an executable.  The problem is
 dnl that the various TRY_LINK / COMPILE_IFELSE macros of Autoconf always
 dnl remove all the temporary files including conftest.o.  So the trick here
 dnl is to temporarily change the value of ac_objext so that conftest.o is
-dnl preserved accross tests.  This is obviously fragile and I will burn in
+dnl preserved across tests.  This is obviously fragile and I will burn in
```
Comment (pieterlexis) [inline m4/boost.m4]: We pull this file from another repo (github.com/tsuna/boost.m4) every now and again, so this change will be reverted the next time we do.

File: codedocs/doxygen.conf [hunk 4]
```diff
@@ -412,7 +412,7 @@ WARN_IF_UNDOCUMENTED   = YES
 
 WARN_IF_DOC_ERROR      = YES
 
-# This WARN_NO_PARAMDOC option can be abled to get warnings for 
+# This WARN_NO_PARAMDOC option can be enabled to get warnings for
```
Comment (Habbie) [inline codedocs/doxygen.conf]: Wonder if doxygen upstream fixed this error since 15 years ago ;)

**Pull request merged**